Caboodle/Caboodle.csproj

```diff
@@ -58,4 +58,4 @@
     <Compile Include="**\*.ios.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildSDKExtrasTargets)" Condition="Exists('$(MSBuildSDKExtrasTargets)')" />
-</Project>
+</Project>
```

Comment:
Please revert these changes to the .csproj

Looks like a tab change and then some newline stuff crept in.